presto-main/src/main/java/io/prestosql/sql/planner/plan/ApplyNode.java

```diff
@@ -101,7 +102,8 @@ private static boolean isSupportedSubqueryExpression(Expression expression)
     {
         return expression instanceof InPredicate ||
                 expression instanceof ExistsPredicate ||
-                expression instanceof QuantifiedComparisonExpression;
+                expression instanceof QuantifiedComparisonExpression ||
+                expression instanceof ArrayConstructor;
```

Comment:
Unclear to me how I'll use this. Will I use this to pattern match my rules?

Reply:
This is mostly for sanity checking. It's to make sure we don't inadvertently create Apply nodes with expressions that are not supported.